kolibri/utils/main.py

```diff
@@ -185,6 +186,27 @@ def _copy_preseeded_db(db_name, target=None):
             )
 
 
+def sqlite_check_foreign_keys():
+    db_connection = connections["default"]
```

Comment:
This is explicitly being done in the method that is before Django setup, so I think using the django DB connection here might not be a good idea. Could you instead generate the connection directly using the Python sqlite library?

We may also want to iterate over all the SQLite  databases listed in https://github.com/learningequality/kolibri/blob/develop/kolibri/deployment/default/sqlite_db_names.py#L14 to be safe?

Reply:
argg, got it. Thanks for the advice
Changed as requested